src/main/java/com/google/songgame/servlets/ChatServlet.java

```diff
@@ -146,13 +148,14 @@ private void markUserGuessedCorrectly(String userId, EmbeddedEntity currentRound
   private boolean checkIfCorrectGuess(String message, EmbeddedEntity currentRound) {
```

Comment:
For the future, it would be good to add a test for this method, especially to make sure we change the message to lower case before comparing it to the video title. Something to consider if you have time later.